### DISCUSSION TITLE
Sections feature

### DISCUSSION THREAD

**Discussion by Zabooby** [excalith/excalith-start-page#36] [Ideas]
Would you be interested in adding the following; clicking the section title opens all the links in the section. 

For example, I have a web comics section which has links that I read all together, rather than having to open each individually, clicking the title would be much faster. 

Basically how Fluidity implements it

**Comment by excalith** [comment]
Hey @Zabooby, I am thinking about implementing a tab system just like in terminals. 

Here is an example of what I meant. The `start-page` on bottom-left is a tab, and whenever you create a new tab, it stacks to the right. 
![Screenshot 2023-05-03 152612](https://user-images.githubusercontent.com/9248355/235915168-ef763bf9-6c0f-455e-bb9e-0934dace725f.png)

The way I see this feature is, you can optionally create categories for each tab and they will display within multiple groups. If you prefer no tabs, it should work as it is right now. Depending on your OS, you should be able to use keybindings such as `CMD + [NUM]` to switch between each tab fast. One downside of this is it should filter possible matches through the active tab, and you will miss the launching filtered links faster (will require switching to corresponding tab first). That is why I think it should be an optional feature.

Such an improvement would be great, but will require a lot of work. I will get into that whenever I find time to do so, but unfortunately I cannot give you an estimation at the moment.

**Reply by Zabooby** [reply]
Interesting, can't wait to try it out!
